routerDestinations/marketo/transform.js

```diff
@@ -0,0 +1,7 @@
+const process = input => {
```

Comment:
We should put this logic in `v0/destinations/marketo/transform.js` in a different method as the code in that file will be reused